src/components/core/popoverMenu/popoverItem.vue

```diff
@@ -31,7 +31,17 @@
 		</a>
 		<!-- If item.input is set instead, an put will be used -->
 		<span v-else-if="item.input" class="menuitem">
-			<input :id="key" :type="item.input" :class="item.input"
+			<!-- does not show if input is checkbox -->
+			<span v-if="item.input != 'checkbox'" :class="item.icon" />
```

Comment:
We need radio as well ;)
But I did not checked how we can do that with radios actually.
Because the inputs then needs to be all in the same form and I don't know how we could manage a form across multiple popover items :thinking: 
Any ideas?

Reply:
hmm good point, it seems something that becomes more clear when we have a use case, for example with the text. Is there something from the docs/ original app I could try recreate to work this out?

Reply:
You're right! Let's support only checkbox and text for now!! :+1: :1st_place_medal: :D